cachalot/tests/tests_decorators.py

```diff
@@ -0,0 +1,44 @@
+import logging
```

Comment:
@Fasther: wouldn’t name file `tests_decorators.py` as it suggests that it contains tests. Better go with `decorators.py`.

Reply:
@dbartenstein I did it on purpose - all tests are in files without "tests" but the utils live in "tests_utils" where are no tests :)